src/ray/object_manager/plasma/create_request_queue.cc

```diff
@@ -93,6 +93,15 @@ Status CreateRequestQueue::ProcessRequests() {
     bool spilling_required = false;
     auto status =
         ProcessRequest(/*fallback_allocator=*/false, *request_it, &spilling_required);
+
+    // if allocation failed due to OOM, and fs_monitor_ indicates the local disk is full,
```

Comment:
Can we test this codepath in create_request_queue_test.cc? And check that we do not spill?